docs/installation/ansible-galaxy.rst

```diff
@@ -0,0 +1,20 @@
+Ansigle Galaxy Installation
+===========================
+
+To have fully installable Pulp3 it is needed to:
+
+1. Ensure python3, ansible( >2.2), libselinux-python and virtualenv are installed
+
+   sudo dnf install python3 ansible libselinux-python
+   sudo pip3 install virtualenv
+
+2. Selinux should be set in permissime mode
```

Comment:
maybe change this to: all target machines should have SELinux set to the permissive mode